800.renames-and-merges/p.yaml

```diff
@@ -356,6 +356,7 @@
 - { setname: protonmail-bridge,        name: protonmail-bridge-bin, addflavor: bin }
 - { setname: protontricks,             name: "python:protontricks" }
 - { setname: protonvpn-cli,            name: [protonvpn-cli-2, protonvpn-linux-cli] }
+- { setname: prowlarr,                 name: [$0-develop, $0-develop-bin, $0-nightly-bin], weak_devel: true, nolegacy: true }
```

Comment:
https://repology.org/projects/?search=prowlarr
there is no `prowlarr-develop-bin` and no `prowlarr-nighly-bin`